validation.go

```diff
@@ -79,7 +85,7 @@ func (v *Validator) CalcRoot(stopAtLayer uint) ([]byte, []ParkingSnapshot, error
 		if activePos.Height == stopAtLayer {
 			break
 		}
-		// The activeNode's sibling should be calculated iff it's an ancestor of the next proven leaf. Otherwise, the
+		// The activeNode's sibling should be calculated if it's an ancestor of the next proven leaf. Otherwise, the
```

Comment:
I think this was intended. Iff stands for "if and only if".

Reply:
hm... golangci-lint marks it as a typo 🤔